website/content/en/highlights/2021-12-14-improved-concurrency-model.md

```diff
@@ -0,0 +1,34 @@
+---
+date: "2021-12-14"
+title: "Improved concurrency model"
+description: "New parallel processing with improved performance"
+authors: ["barieom"]
+pr_numbers: [10265]
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+We've released an improved concurrency model that provides measurable performance
+improvements and enables more efficient vertical scaling.
+
+Previously, Vector set the limit to active tasks to the number of available CPUs
+in a given environment. Transforms were run sequentially, which made transforms
+a bottleneck in Vector pipelines.
+
+With this new release, Vector will perform faster when a transform is a bottleneck,
+assuming that more CPUs are available to share their work. This works by Vector
+determine whether an individual transform will be processed in parallel when there
+is a sufficient load to the environment.
```

Comment:
@lukesteensen would it be worth noting in the docs which transforms support parallelization and which don't?

Reply:
Yeah we could do that. The current list is:
```
src/transforms/tokenizer.rs
src/transforms/regex_parser.rs
src/transforms/metric_to_log.rs
src/transforms/filter.rs
src/transforms/grok_parser.rs
src/transforms/key_value_parser.rs
src/transforms/json_parser.rs
src/transforms/logfmt_parser.rs
src/transforms/log_to_metric.rs
src/transforms/remap.rs
src/transforms/aws_cloudwatch_logs_subscription_parser.rs
```

Reply:
I folded this list into the article!

Reply:
We can probably leave out the deprecated transforms from this list. I'll trim it down.

In my original comment, I was thinking about having this be a cue metadata field to be rendered on the website though. Do you think something like that would be reasonable?

Reply:
I'm not sure about it being a metadata field in cue. It feels a little bit like an implementation detail to me, but I'm also not the target audience for that data. I don't have a strong opinion either way.

Reply:
I agree, we don't need communicate this to users in that way. Concurrency should "just work".